### PR TITLE
Added support for iPhone 6s, iPhone 6s Plus and iPhone 6plus

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
Added support for iPhone 6s, iPhone 6s Plus and iPhone 6plus to fix https://github.com/appium/appium-dot-app/issues/514. please review the same.

### CODE REVIEW

File: lib/devices/ios/ios.js [hunk 1]
```diff
@@ -1036,6 +1036,18 @@ IOS.getDeviceStringFromOpts = function (opts) {
       'iPad Simulator (9.0)': 'iPad 2 (9.0)',
       'iPad Simulator (9.1)': 'iPad 2 (9.1)',
       'iPad Simulator (9.2)': 'iPad 2 (9.2)',
+      'iPhone Simulator (8.1)': 'iPhone 6s (8.1)',
```
Comment (imurchie) [inline lib/devices/ios/ios.js]: This hash just specifies the default, so each type should only be in once.

iPhone Simulator for 8.1-8.4 are already defined above.

Comment (SrinivasanTarget) [inline lib/devices/ios/ios.js]: Yeah but it is not listed for iPhone 6S,6S plus and 6plus devices right? Please clarify.

Comment (imurchie) [inline lib/devices/ios/ios.js]: That does not matter. For a single key, like `iPhone Simulator (8.1)`, we only want to map to a single device, like `iPhone 6 (8.1)`. For you to be able to use, say, `iPhone 6s Plus` in 9.1, there needs to exist an entry on the left side, for `iPhone 6s Plus (9.1)`, that maps to a valid device. But each key (i.e., left side of the "x: y" assignment) needs to be **unique**.

**Comment by SrinivasanTarget** [comment]
@imurchie  yeah got it now.Removed the redefined simulator listings for 8.1 - 8.4.

**Comment by jlipps** [comment]
LGTM, isaac does this need to be ported to appium 1.5?

**Comment by imurchie** [comment]
No, it is already implemented there (https://github.com/appium/appium-ios-simulator/blob/master/lib/simulator-xcode-7.js#L32)

File: lib/devices/ios/ios.js [hunk 2]
```diff
@@ -1044,7 +1044,25 @@ IOS.getDeviceStringFromOpts = function (opts) {
...
       'iPhone 6 (9.1)': 'iPhone 6 (9.1) [',
-      'iPhone 6 (9.2)': 'iPhone 6 (9.2) ['
+      'iPhone 6 (9.2)': 'iPhone 6 (9.2) [',
+      'iPhone Simulator (9.0)': 'iPhone 6s (9.0) [',
+      'iPhone Simulator (9.1)': 'iPhone 6s (9.1) [',
+      'iPhone Simulator (9.2)': 'iPhone 6s (9.2) [',
+      'iPhone 6s (9.0)': 'iPhone 6s (9.0) [',
+      'iPhone 6s (9.1)': 'iPhone 6s (9.1) [',
+      'iPhone 6s (9.2)': 'iPhone 6s (9.2) [',
+      'iPhone Simulator (9.0)': 'iPhone 6 Plus (9.0) [',
```
Comment (imurchie) [inline lib/devices/ios/ios.js]: Still a repeat.

File: lib/devices/ios/ios.js [hunk 3]
```diff
@@ -1044,7 +1044,25 @@ IOS.getDeviceStringFromOpts = function (opts) {
...
+      'iPhone Simulator (9.1)': 'iPhone 6 Plus (9.1) [',
+      'iPhone Simulator (9.2)': 'iPhone 6 Plus (9.2) [',
+      'iPhone 6 Plus (9.0)': 'iPhone 6 Plus (9.0) [',
+      'iPhone 6 Plus (9.1)': 'iPhone 6 Plus (9.1) [',
+      'iPhone 6 Plus (9.2)': 'iPhone 6 Plus (9.2) [',
+      'iPhone Simulator (9.0)': 'iPhone 6s Plus (9.0) [',
```
Comment (imurchie) [inline lib/devices/ios/ios.js]: Same.

Comment (SrinivasanTarget) [inline lib/devices/ios/ios.js]: Hope now it is perfect.

**Comment by imurchie** [comment]
Excellent! This is correct now. 

One last step: can you squash this into a single commit?

**Comment by SrinivasanTarget** [comment]
@imurchie Its Squashed to one now.

**Comment by moizjv** [comment]
:+1:

**Comment by imurchie** [comment]
Thanks!

**Pull request merged**